css/style.css

```diff
@@ -27,14 +27,23 @@ body{
 .play{background: url('../img/play.png') ;}
 .pause{background: url('../img/pause.png') ;}
 
+#track-name{
```

Comment:
Well, I used bootstrap here and also width and height are auto, so they will automatically order themselves.